src/components/tag-skill/tag-skill.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import iconTick from '../../svgs/icon-tick.svg';
+import Icon from '../icon/icon.jsx';
+import Tag from '../tag/tag.jsx';
+import styles from './tag-skill.css';
+
+export default function TagSkill(props) {
+  return (
+    <Tag readOnly>
+      <span className={styles.row}>
+        {props.name}
+        <span className={styles.level + (props.level ? '' : ` ${styles['level-tick']}`)}>
```

Comment:
If the `level-tick` class composes from `level` then we don't need string concatenation. 

```suggestion
        <span className={props.level ? styles.level : styles['level-tick']}>
```

Way easier to read!

It can also help us identify how to reduce some overhead. Perhaps this is even easiest to read: 

```
{props.level ? (
  <span className={styles.level}>{props.level}</span>
) : (
  <span className={styles['level-tick']}>
    <Icon name={iconTick.id} size="small" stroke="skip" fill="skip" currentColor="skip" />
  </span>
)}
```